### ISSUE TITLE
Habitica home page crashes on startup in IE11

### ISSUE THREAD

**Issue by Zian** [HabitRPG/habitica#10919]
Steps to reproduce:
# Launch IE11
# Go to https://habitica.com/

The IE11 developer console says:

DOM7011: The code on this page disabled back and forward caching. For more information, see: http://go.microsoft.com/fwlink/?LinkID=291337
File: habitica.com
HTML1300: Navigation occurred.
File: habitica.com
HTML1500: Tag cannot be self-closing. Use an explicit closing tag.
File: habitica.com, Line: 1, Column: 1800
SCRIPT1046: Multiple definitions of a property not allowed in strict mode
File: app.dcef6e40a692ebbd4efa.js, Line: 1, Column: 971040

**Comment by Zian** [comment]
Should be:

1. Launch IE11
2. Go to https://habitica.com/

**Comment by Alys** [comment]
Thanks for providing all those details! This looks like the first important error from the console output:
```
HTML1500: Tag cannot be self-closing. Use an explicit closing tag.
File: habitica.com, Line: 1, Column: 1800
```

Assuming IE11 is getting the same code as my Firefox browser, then column 1800 refers to the start of the `path` element that's shown in the view-source output below (I've added line breaks for clarity):
```
<!DOCTYPE html><html><head><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1"><title>Habitica - Gamify Your Life</title><meta name=description content="Habitica is a free habit and productivity app that treats your real life like a game. Habitica can help you achieve your goals to become healthy and happy."><meta name=keywords content=Habits,Goals,Todo,Gamification,Health,Fitness,School,Work><meta name=smartbanner:title content=Habitica><meta name=smartbanner:author content="HabitRPG, Inc."><meta name=smartbanner:price content=FREE><meta name=smartbanner:price-suffix-apple content=" - On the App Store"><meta name=smartbanner:price-suffix-google content=" - In Google Play"><meta name=smartbanner:icon-apple content=/static/presskit/Logo/iOS.png><meta name=smartbanner:icon-google content=/static/presskit/Logo/Android.png><meta name=smartbanner:button content=VIEW><meta name=smartbanner:button-url-apple content=https://itunes.apple.com/us/app/habitica-gamified-taskmanager/id994882113><meta name=smartbanner:button-url-google content="https://play.google.com/store/apps/details?id=com.habitrpg.android.habitica"><meta name=smartbanner:enabled-platforms content=android,ios><link href="https://fonts.googleapis.com/css?family=Roboto+Condensed:400,400i,700,700i|Roboto:400,400i,700,700i" rel=stylesheet><link rel="shortcut icon" sizes=48x48 href=/static/icons/favicon.ico><link rel="shortcut icon" sizes=192x192 href=/static/icons/favicon_192x192.png><link rel=mask-icon href=/static/icons/favicon.ico><meta property=og:image content=/static/emails/images/meta-image.png><link href=/static/css/app.0001e145b30569e686be39f55dd2a7a9.css rel=stylesheet></head><body><div id=loading-screen><svg id=melior xmlns=http://www.w3.org/2000/svg viewBox="0 0 61.91 64">

<path d=M61.82,64H51.59c-3.08,0-3.72.37-3.67-1,0.07-1.87.67-1.94,2.63-2.49,1.63-.45,1-3.35-0.8-5.88-1.28-1.76-3.89-3.81-7.31-2.22a10.75,10.75,0,0,0-4.56,3.52c-1.68,2.33-1.59,4.54,1,4.54s5.39-1.5,6.23.64c1,2.64.33,2.89-.18,2.89H28.55v0C19.77,64,11,63.93,9,58.38c-2.82-7.68,7.43-10.64,7.75-15.46,0.13-2-1-2.85-2.34-2.85h-6V36.41H4.7v-11H8.36V29.1H12v3.65h3.65v5.08a5.76,5.76,0,0,1,3.07,5.05c-0.17,5.51-9.5,8.57-7.79,14.35,1.56,5.29,13.37,4,13,.74L23.7,56.1c-0.06-2.62-.47-6.12.08-9.22C24.64,42,27.67,37.78,33,37.74c1,0,1.78-.21,1.78-1s-1.55-.84-2.64-0.95a23.35,23.35,0,0,1-12.56-5c-2.43-2-6.21-8.3-3.74-7.83a21.74,21.74,0,0,0,4.06.4c1.24,0,4.44-.35,4.44-1.11,0-1-1.85-.42-4.57-0.68C16.48,21.22,9.6,19.83,6,9.35,4.71,5.43,3.83-1.91,6,.46c12.46,13.7,16.69,11.47,23.84,16.16,3.15,2.06,5.19,7,7,6.58,1.2-.27.46-1.37,0.64-3.93C37.66,17,38.75,16.48,36,15.79c-3.26-.81-6.52-4.38-4.39-4.33a11.89,11.89,0,0,0,5.53-.76c1.87-.81,6.43-4.28,9.18-2.89s5.08-.6,6.94-0.25c2.71,0.51,3.41,4.24,3.05,6.42-0.22,1.38-.22,1.38-2,1.28-3.61-.21-4.53,2.67-2,4.25,3.87,2.42,5.51,4.23,6.56,9.58,0.51,2.6.1,3.2-.76,2.72s-2.34-.72-0.29,4-1.29,10.28-2.39,10.9a1.3,1.3,0,0,0-.91,1.34c0,11.42,0,12.27,1.92,12.48,2.9,0.31,4.14-1.44,5.27.06C63.29,62.73,63.41,64,61.82,64ZM4.7,21.28H1v3.65H4.7V21.28Z transform=translate(-1.05) fill=#fff />

</svg></div><div id=app></div><script type=text/javascript src=//cloudfront.loggly.com/js/loggly.tracker-latest.min.js async></script><script type=text/javascript src=/api/v4/i18n/browser-script></script><script type=text/javascript src=/static/js/manifest.1c248b22870217f28c30.js></script><script type=text/javascript src=/static/js/vendor.6c4ff781be5de0a8572b.js></script><script type=text/javascript src=/static/js/app.dcef6e40a692ebbd4efa.js></script></body></html>
```

While it's true that that tag is self-closed, the W3C specification allows that, as shown by the examples at https://www.w3.org/TR/SVG/paths.html#PathElement
I think this is another case of IE not following internet standards. It's several years old now so it's not surprising that it's having problems.

Does the website work on Edge on the same machine? If you can't use Edge on that machine, does the site work on Firefox or Chrome or any browsers based on them?

**Comment by paglias** [comment]
I think the issue comes from the second error 

```
 SCRIPT1046: Multiple definitions of a property not allowed in strict mode
File: app.dcef6e40a692ebbd4efa.js, Line: 1, Column: 971040
```

I can't use ie11 but if someone can it would be great if they could click on that error and see if a more precise location is given in the source map

**Comment by Zian** [comment]
I am using Windows 7 so I can't try Edge.

The website works in Firefox.

**Comment by paglias** [comment]
@Zian I've found the source of the problem and working on a fix

**Comment by paglias** [comment]
@Zian this has been fixed, can you tell us if it's working properly now?

**Comment by citrusella** [comment]
I'm not Zian but I tried IE11 out and saw the same bug when I saw this earlier, but it works now (other than some visual bugs (...should these be specifically reported in their own issue? are they already reported?), like the skills window being part offscreen (and the skill titles not being constrained to the white half), the g1g1 banner (but not the inn banner) and several parts of the interface taking up strange amounts of space (usually extra space)).

(The actual hang issue talked about here is *not* fixed on the delta site, fwiw, but I think that's just the change not having been pushed to it. XP)

I'm on Windows 7, myself, though I'm not sure my version of IE11 is the latest version (the last time I manually updated it, it seemed to go through all the motions of installing the new version but failed to update the version number in the about screen which makes me think the whole update failed somehow). Moot point considering the age of the browser, probably (and the health of my own version, which has had issues since an encounter with a trojan a few years back, around the time I stopped using it because of some issues that cropped up due to that).
